Makefile

```
@@ -2071,10 +2071,12 @@ ocamltest/ocamltest.html: ocamltest/ocamltest.org
 
 .PHONY: otherlibraries
 otherlibraries: ocamltools
+	$(MAKE) dynlink-all
 	$(MAKE) -C otherlibs all
 
 .PHONY: otherlibrariesopt
 otherlibrariesopt:
+	$(MAKE) dynlink-allopt
```

No need for the recursive call to make (I checked locally):
```suggestion
otherlibraries: ocamltools dynlink-all
	$(MAKE) -C otherlibs all

.PHONY: otherlibrariesopt
otherlibrariesopt: dynlink-allopt
```

---

Thanks, committed!